lib/ay_tests/installer.rb

```diff
@@ -217,7 +221,21 @@ def zypper_install(packages)
         "--auto-agree-with-licenses",
         "--name"
       ]
-      Cheetah.run(parts + Array(packages))
+      candidates = Array(packages)
+      to_install = candidates.select do |name|
+        begin
+          Cheetah.run "rpm", "-qi", name
```

Comment:
I just wanted to check if the package is installed. The installation is done with zypper.